js/src/carousel.js

```diff
@@ -409,6 +411,21 @@ class Carousel {
     }
   }
 
+  _updateInterval() {
+    const element = this._activeElement || SelectorEngine.findOne(SELECTOR_ACTIVE_ITEM, this._element)
+
+    if (element) {
```

Comment:
just a small change, but I prefer if we can return asap so this check should be:

```js
if (!element) {
  return
}